shares.tf

```diff
@@ -32,3 +32,19 @@ resource "openstack_sharedfilesystem_share_access_v2" "home_share_access_rw" {
   access_level = "rw"
 }
 
+resource "openstack_sharedfilesystem_share_v2" "data_share" {
```

Comment:
I was just wondering how the /data mount point would know what it's size would be etc.

Reply:
yeah, actually should be in a variable!